modules/Vtiger/ListView.php

```diff
@@ -272,6 +272,12 @@
 }
 $smarty->assign('advancedSearchMapResult', $advancedSearchMapResult);
 
+// GV responsible for showing or hiding filter panel
+$Application_ListView_FilterPanel_Open = GlobalVariable::getVariable('Application_ListView_FilterPanel_Open', '1', $currentModule, '', $_REQUEST['action']);
+if (!$Application_ListView_FilterPanel_Open) {
+	$smarty->assign('Application_ListView_FilterPanel_Open', "display: none;");
+}
```

Comment:
you need an "else" here to set the Smarty variable in the case it is not open

Reply:
the reason why I kept it like that is because Smarty did not crash .. but I will change it right away